modules/ps_customersignin/ps_customersignin.tpl

```diff
@@ -59,5 +59,19 @@
     {/if}
     {hook h='displayTtCompareHeader'}
     {hook h='displayTtWishlistHeader'}
+    
+      <!-- FB sign up button -->
+    
+    <li>
+      <a onclick="fb_login();" class="profile-btn onclick-btn" title="{l s='Sign in with Facebook' mod='hifacebookconnect'}">
+       Sign in with 
+       <svg width="14" height="14" viewBox="0 0 18 34" fill="none" xmlns="http://www.w3.org/2000/svg">
+       <path d="M14.6624 5.64542H17.7663V0.239417C17.2308 0.16575 15.3891 0 13.2443 0C3.42398 0 6.09581 11.1208 5.70481 12.75H0.764893V18.7935H5.70339V34H11.7582V18.7949H16.497L17.2492 12.7514H11.7568C12.0231 8.75075 10.6787 5.64542 14.6624 5.64542Z" fill="white"/>
+       </svg>
+  </a>
+    </li>
+    
+      <!-- FB sign up button -->

```

Comment:
Add `<!-- /end FB sign up button -->` because of where block end before we relocate it